variables.tf

```
@@ -216,6 +216,12 @@ variable "identity_ids" {
   default     = []
 }
 
+variable "http2_enabled" {
+  description = "Specifies if the HTTP2 protocol should be enabled."
```

Make description more similar to existing variables:

```suggestion
  description = "Should the HTTP2 protocol be enabled for this Function App?"
```